### DISCUSSION TITLE
Error on fresh install - "exports is not defined"

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [nuxt/nuxt#18227] [Questions] [answered]
On fresh initialization of nuxt 3 project, getting error as shown below

![image](https://user-images.githubusercontent.com/11911938/139790377-d3b1c098-315d-47ca-bf23-8aca663bc3e6.png)

Steps to reproduce
```
npx nuxi init nuxt3-app
cd nuxt3-app
npm i
npm run dev
```

My Setup
Node: `14.16.1`
Npm: `8.1.2`
Windows 10 x64

When I tried the same after installing  node `17.0.1` the error is "exports is not defined in ES module scope"
![image](https://user-images.githubusercontent.com/11911938/139791209-b0725467-29d0-4a1c-9056-dee9e7f73bf3.png)

**Comment by danielroe** [comment, marked as answer]
Closed in https://github.com/unjs/mlly/pull/21. Next time, would you create an issue for a bug rather than a discussion?
